src/main/java/kg/jarkyn/server/Resource/PublicResource.java

```diff
@@ -0,0 +1,21 @@
+package kg.jarkyn.server.Resource;
+
+import java.io.File;
+import java.nio.file.Files;
+
+public class PublicResource {
```

Comment:
I did mention that the feedback you guys previously give has not yet been applied, this is one of those classes, will definitely clean this up, responsibilities are not clear indeed